Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt

```diff
@@ -192,15 +195,14 @@ private fun MapViewContainer(
     latitude: String,
     longitude: String
 ) {
+    val cameraPosition = remember(map) { LatLng(latitude.toDouble(), longitude.toDouble()) }
```

Comment:
This is a good opportunity to showcase `rememberSaveable`?

Reply:
That'd be a micro-optimization IMO, as this is not a state to remember, just a constant I don't want to compute multiple times.